system-addon/common/PrerenderData.jsm

```diff
@@ -52,6 +52,9 @@ this.PrerenderData = new _PrerenderData({
     "showTopSites": true,
     "showSearch": true,
     "topSitesCount": 6,
+    "collapseTopSites": false,
+    "section.highlights.collapsed": false,
+    "section.topstories.collapsed": false,
```

Comment:
I should probably change all these to be one pref that's an array of section IDs 🤔